resources/app/src/components/expert_panels/ReviewMembership.vue

```diff
@@ -1,178 +1,172 @@
 <script setup>
-import { computed, ref, watch } from 'vue'
-import axios from 'axios'
-import { hasPermission } from '../../auth_utils';
+import { computed, ref, watch, onMounted } from 'vue';
 import CredentialsView from '../people/CredentialsView.vue';
 import ExpertisesView from '../people/ExpertisesView.vue';
-import { formatDate } from '@/date_utils'
 
 const props = defineProps({
-    members: {
-        required: true,
-        type: Array
-    },
+    members: { required: true, type: Array },
+    type: { required: true, type: String }
```

Comment:
You've added a `type` prop here, but not committed the changes to files using this prop, so when I run the code I get errors related to this.